tox/session.py

```diff
@@ -133,7 +134,16 @@ def _initlogpath(self, actionid):
         f.flush()
         return f
 
-    def popen(self, args, cwd=None, env=None, redirect=True, returnout=False, ignore_ret=False):
+    def popen(
+        self,
+        args,
+        cwd=None,
+        env=None,
+        redirect=True,
+        returnout=False,
+        ignore_ret=False,
+        shell=False,
```

Comment:
I'm actually pretty `-1` on this.  shells are a great way to expose a CVE.  If someone wants a shell they can already `['bash', '-c', ...]`